tools/arm_pack_manager/index.json

```diff
@@ -462354,6 +462354,564 @@
+        },
+        "algorithms": [
+          {
+            "file_name": "CMSIS/Flash/STM32WB_M4.FLM",
+            "start": 134217728,
+            "size": 262144,
+            "default": true,
+            "ram_start": null,
+            "ram_size": null
+          }
+        ],
+        "processor": {
+          "Symmetric": {
+            "units": 1,
+            "core": "CortexM4",
+            "fpu": "SinglePrecision",
+            "mpu": "Present"
+          }
+        },
+        "from_pack": {
+          "vendor": "Keil",
+          "pack": "STM32WBxx_DFP",
+          "version": "1.0.0",
+          "url": "http://www.keil.com/pack"
+        },
+        "vendor": "STMicroelectronics:13",
+        "family": "STM32WB Series",
+        "sub_family": "STM32WB55"
+      },
+      "STM32WB55REVx": {
+        "name": "STM32WB55REVx",
+        "memories": {
+          "IRAM1": {
+            "access": {
+              "read": true,
+              "write": true,
+              "execute": false,
+              "peripheral": false,
+              "secure": false,
+              "non_secure": false,
+              "non_secure_callable": false
+            },
+            "start": 536870912,
+            "size": 262144,
+            "startup": false,
+            "default": true
+          },
+          "IROM1": {
+            "access": {
+              "read": true,
+              "write": false,
+              "execute": true,
+              "peripheral": false,
+              "secure": false,
+              "non_secure": false,
+              "non_secure_callable": false
+            },
+            "start": 134217728,
+            "size": 524288,
+            "startup": true,
+            "default": true
+          }
+        },
+        "algorithms": [
+          {
+            "file_name": "CMSIS/Flash/STM32WB_M4.FLM",
+            "start": 134217728,
+            "size": 524288,
+            "default": true,
+            "ram_start": null,
+            "ram_size": null
+          }
+        ],
+        "processor": {
+          "Symmetric": {
+            "units": 1,
+            "core": "CortexM4",
+            "fpu": "SinglePrecision",
+            "mpu": "Present"
+          }
+        },
+        "from_pack": {
+          "vendor": "Keil",
+          "pack": "STM32WBxx_DFP",
+          "version": "1.0.0",
+          "url": "http://www.keil.com/pack"
+        },
+        "vendor": "STMicroelectronics:13",
+        "family": "STM32WB Series",
+        "sub_family": "STM32WB55"
+      },
+      "STM32WB55RGVx": {
+        "name": "STM32WB55RGVx",
+        "memories": {
+          "IROM1": {
+            "access": {
+              "read": true,
+              "write": false,
+              "execute": true,
+              "peripheral": false,
+              "secure": false,
+              "non_secure": false,
+              "non_secure_callable": false
+            },
+            "start": 134217728,
+            "size": 16777216,
```

Comment:
There is an issue (already reported to CMSIS ST team) with ROM size value,
Is it possible to set 1M instead of 16M ? (1048576 instead of 16777216)

Reply:
It's likely wrong in the original CMSIS-pack itself then?
It should be fixed there, too.

Reply:
Yes, already reported a long time ago.... :-(
Don't understand yet if it is Keil or ST side...

Reply:
@jeromecoutant Send me an email with details. I would assume the pack belongs to ST but if that is not the case, lets find out and have a fix there. I can  talk to CMSIS team

Reply:
This page at least does also advertise it as 16 MB? 
https://www.keil.com/dd2/stmicroelectronics/stm32wb55rgvx/
If I'm hitting the right line... 

```
      "STM32WB55RGVx": {
        "name": "STM32WB55RGVx",
        "memories": {
          "IROM1": {
            "access": {
              "read": true,
              "write": false,
              "execute": true,
              "peripheral": false,
              "secure": false,
              "non_secure": false,
              "non_secure_callable": false
            },
            "start": 134217728,
            "size": 16777216,
            "startup": true,
            "default": true
```
Seems to be for some reason line 466474 in my editor.

Reply:
However, in this page; https://www.digikey.co.uk/product-detail/en/stmicroelectronics/STM32WB55RGV6/497-18488-ND/9919553?utm_campaign=buynow&utm_medium=aggregator&WT.z_cid=ref_findchips_standard&utm_source=findchips - it is indeed listed as 1 MB part, so I think the source information going to Keil has to be wrong. But, we can fix it here.

Reply:
Fixed now.